aip/auth/4122.md

```diff
@@ -0,0 +1,349 @@
+---
+id: 4122
+scope: auth
+state: reviewing
+created: 2020-12-15
+---
+
+# Service Account Impersonation
+
+A service account or a user identity in GCP can impersonate another service 
+account, and request short-lived credentials on behalf of that service account.
+
+The request can be direct or delegated.
+
+In the direct case, the caller is authenticated as either a Google account or a
```

Comment:
Is "Google account" the same thing as "user identity"? If they mean the same thing, let's use just one term for that concept.